Seed Phrase Flow

BenJamminnn opened the pull request:
# Seed Phrase Flow 

- Establish Seed Phrase flow outlined in Figma
- Modules for showing Seed Phrase and Testing Seed Phrase
- Add `AccessibilityIdentifier` file to house all accessibility ids in one place

## Screenshots 


https://github.com/LibertyDSNP/dsnp-wallet-swift/assets/5061628/4256a625-9845-43cb-9749-6e471fe9a403


https://github.com/LibertyDSNP/dsnp-wallet-swift/assets/5061628/c61a3775-4c84-4cf4-b73f-f5bec2bf3738

BenJamminnn committed:
add new fonts

BenJamminnn committed:
update bg color and fonts in labels

BenJamminnn committed:
view updates

BenJamminnn committed:
swift ui view for seed phrase + boilerplate

BenJamminnn committed:
seed phrase view work - wip

BenJamminnn committed:
fix seed phrase view

BenJamminnn committed:
seed phrase test view work, logic for passing test - wip

BenJamminnn committed:
fix puzzle `isCorrect` logic, wip

BenJamminnn committed:
add puzzle element, font, design updates, use dummy data

BenJamminnn committed:
build word bank ui

BenJamminnn committed:
seed phrase test logic, view updates - wip

BenJamminnn committed:
add navigation, view updates - wip

BenJamminnn committed:
seed puzzle logic - wip

BenJamminnn committed:
make view model observed

BenJamminnn committed:
fix puzzle logic

BenJamminnn committed:
add continue button, clean up, `isCorrect` logic

BenJamminnn committed:
fix first available index logic issue

BenJamminnn committed:
fix indicies off by one issue

BenJamminnn committed:
fix enabled button function

BenJamminnn committed:
adjust button height + width

BenJamminnn committed:
add titles, shuffled seed phrase, fix continue button logic

BenJamminnn committed:
add accessiblity ids

BenJamminnn committed:
fix logic error

ryansheh commented:
Functionally look's good, but just make sure the UI/UX theming is accurate for QA. From what I can see:
* The alignment in the spec is left-aligned
* Selecting a seed phrase word during the test looks like its bolded in the selected words

Also, 
* Could you delete the old (my) seed phrase logic if it's being unused?

BenJamminnn committed:
delete old seed phrase implementation, make text leading

BenJamminnn commented:
I deleted the old implementation


Updated screenshots without centered elements: 

<img width="312" alt="Screenshot 2023-05-18 at 3 29 20 PM" src="https://github.com/LibertyDSNP/dsnp-wallet-swift/assets/5061628/57002b49-c316-4922-b8ea-340610004da7">
<img width="308" alt="Screenshot 2023-05-18 at 3 29 29 PM" src="https://github.com/LibertyDSNP/dsnp-wallet-swift/assets/5061628/fd7b4360-2f41-4333-979d-befdcb251e70">

BenJamminnn committed:
only index to be bold in puzzle

BenJamminnn commented:
Fixed the Bold word issue: 

<img width="244" alt="Screenshot 2023-05-22 at 9 45 36 AM" src="https://github.com/LibertyDSNP/dsnp-wallet-swift/assets/5061628/32772927-4fbb-4473-b7b3-c41da1a65a04">

BenJamminnn committed:
Merge branch 'dev' into seedPhraseViewController
